### DISCUSSION TITLE
some notes dont have hitsounds

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#17916] [Q&A]
Ive noticed if a note comes imminently after a slider end in 1/8 time, it doesn't have a sound.

the last stream of this map has 2 examples, the 1 and the 5. 

[Duskfox & Vixage - Aurora (Axilotl).zip](https://github.com/ppy/osu/files/8534368/Duskfox.Vixage.-.Aurora.Axilotl.zip)

https://user-images.githubusercontent.com/78441708/164534455-fcc6de63-82dd-429d-9bee-337a4e976b19.mp4

although this happens all throughout the map. 

using arch linux.

**Comment by peppy** [comment]
Can you please confirm whether this only happens in the editor? If so, this is related to frame stable playback and can occur at higher frame times (tracked via https://github.com/ppy/osu/discussions/16335 but probably needs an issue as well)

**Reply by Axilotl17** [reply]
yes, only in editor.

**Reply by Axilotl17** [reply]
and on the
> Resolved with multi threading and vsync fps cap

this didn't fix anything for me.

**Reply by peppy** [reply]
It isn't guaranteed to fix anything, and depends on how fast the frames are being processed by the game. It's also not intended to be a final solution, we will fix this properly at some point.

**Reply by Axilotl17** [reply]
okay, thanks.
